include/net/ethernet.h

```diff
@@ -126,6 +139,10 @@ enum ethernet_hw_caps {
 
 	/** VLAN Tag stripping */
 	ETHERNET_HW_VLAN_TAG_STRIP	= BIT(14),
+
+	/** DSA switch */
+	ETHERNET_DSA_SLAVE_PORT	= BIT(15),
```

Comment:
Indentation seems to be off here

Reply:
After applying the patch it looks to be consistent with the rest of this file (by using TABs).